nix/README.md

```diff
@@ -0,0 +1,102 @@
+# nix
```

Comment:
Hot take: all READMEs should be wrapped at 80 characteres

Reply:
I think it makes it easier to read, and doesn't rely on the reader's editor doing soft-wraps

Reply:
this idea is growing on me, not yet ready to dive in that way